mrp_bom_component_find/README.rst

```diff
@@ -0,0 +1,74 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+====================
+MRP Product Use Case
```

Comment:
Please you have to rename all strings Use Case if useless in your module without forget one (in screenshot too). 
Sorry and thanks